test/integrations/destinations/criteo_audience/dataDelivery/other.ts

```diff
@@ -0,0 +1,257 @@
+import { params, headers } from './business';
+import { generateProxyV1Payload } from '../../../testUtils';
+
+export const v1OtherScenarios = [
+  {
+    id: 'criteo_audience_other_0',
+    name: 'criteo_audience',
+    description: '[Other]:: Test for checking service unavailable scenario',
+    successCriteria: 'Should return a 500 status code with',
+    scenario: 'other',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            method: 'PATCH',
+            endpoint: 'https://api.criteo.com/2022-10/audiences/34897/contactlist',
```

Comment:
Don't use the mocks the `mocks` form `network.ts` for these scenarios use these https://github.com/rudderlabs/rudder-transformer/blob/9e047747a119cd3e23cb0c352363788f40e0ef42/test/integrations/common/network.ts#L1